charts/ark-solr/templates/_tplValues.tpl

```diff
@@ -0,0 +1,8 @@
+
+{{- define "common.tplvalues.render" -}}
+    {{- if typeIs "string" .value }}
+        {{- tpl .value .context }}
+    {{- else }}
+        {{- tpl (.value | toYaml) .context }}
+    {{- end }}
+{{- end -}}
```

Comment:
@naveen-armedia Can we merge this file with `_helpers.tpl`?

Reply:
@fabricetriboix  seperated this helper template so that in future we can reuse it by keeping this file in a separate helm chart. And the dependency in the Chart.yaml.